libtock/hd44780.c

```diff
@@ -0,0 +1,282 @@
+#include "gpio.h"
+#include "hd44780.h"
+#include "tock.h"
+
+#define ALLOW_BAD_VALUE 200
+
+/*
+ * hd44780_start is the first function to be called and initializes the
+ * functioning parameters and communication parameters of the LCD, according
+ * to its datasheet (HD44780).
+ *
+ * Example:
+ *  hd44780_begin(void);
+ *
+ */
+int hd44780_begin(void)
+{
+  int ret = command(DRIVER_HD44780_NUM, 0, 16, 1);
+  if (ret)
+    return TOCK_EBUSY;
+  ret = command(DRIVER_HD44780_NUM, 0, 16, 2);
+  if (ret)
+    return TOCK_EBUSY;
+  return TOCK_SUCCESS;
+}
+
+/* hd44780_clear clears the display and sets the cursor posiziton to zero
+ *
+ * Example:
+ *  hd44780_clear(void);
+ */
+int hd44780_clear(void)
+{
+  int ret = command(DRIVER_HD44780_NUM, 3, 0, 0);
+  if (ret)
+    return TOCK_EBUSY;
+  return TOCK_SUCCESS;
```

Comment:
```suggestion
  return command(DRIVER_HD44780_NUM, 3, 0, 0);
```

Here and everywhere else, what is the purpose for masking all errors to `EBUSY`? Why not report the actual error?

Reply:
Hello, I updated!